### PR TITLE
feat: add files and commit

### PR DESCRIPTION
**Pull request by SudoSurya**
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## Problem/Feature

- Git add files to the repository.
- Git commit the initial files with the message "Initial commit."
## Checklist

- [x] I have self-reviewed the changes being requested
- [ ] I have updated the documentation (if applicable)

### CODE REVIEW

File: .github/workflows/generate-linter.yml [hunk 1]
```diff
@@ -28,6 +27,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Commit report
+        run: |
+          git config --global user.name 'BATMAN'
+          git config --global user.email 'BATMAN69@users.noreply.github.com'
```
Comment (Melkeydev) [inline .github/workflows/generate-linter.yml:33]: What is the BATMAN global username??

File: cmd/program/program.go [hunk 2]
```diff
@@ -431,6 +431,20 @@ func (p *Project) CreateMainFile() error {
...
+        cobra.CheckErr(err)
+        return err
+    }
+    // Git commit files
+    err = utils.ExecuteCmd("git", []string{"commit", "-m", "Initial commit"}, projectPath)
+    if err != nil {
+        log.Printf("Error committing files to git repo: %v", err)
+        cobra.CheckErr(err)
+        return err
+    }
```
Comment (Melkeydev) [inline cmd/program/program.go:447]: So will these fire off right away? How would it know the remote branch?

File: .github/workflows/generate-linter.yml (hunk 1 above)
Comment (SudoSurya) [reply .github/workflows/generate-linter.yml:33]: It's for the linter to pass the checks.

File: cmd/program/program.go (hunk 2 above)
Comment (SudoSurya) [reply cmd/program/program.go:447]: We are not pushing the code . We just make an initial commit for users to have an initial snapshot of repo.

**Review by Melkeydev — APPROVED** [review]
LGTM

**Pull request merged**